lldb/include/lldb/Symbol/DWARFCallFrameInfo.h

```diff
@@ -128,8 +128,14 @@ class DWARFCallFrameInfo {
 
   void GetFDEIndex();
 
-  bool FDEToUnwindPlan(dw_offset_t offset, Address startaddr,
-                       UnwindPlan &unwind_plan);
+  // Parsed representation of a Frame Descriptor Entry.
```

Comment:
Think of the ~children~Doxygen.
```suggestion
  /// Parsed representation of a Frame Descriptor Entry.
```